From f63bce5f73492ab6cb06beb04138677447f6cda0 Mon Sep 17 00:00:00 2001
From: Nicolas Ferre <nicolas.ferre@atmel.com>
Date: Fri, 10 Jun 2011 17:21:28 +0200
Subject: [PATCH 055/319] media/video: atmel-isi: add dumb set_parm()

Add dumb set_parm() & get_parm() function to struct soc_camera_host_ops.
Needed for ffmpeg to be able to capture frames from ISI driver.

Signed-off-by: Nicolas Ferre <nicolas.ferre@atmel.com>
---
 drivers/media/video/atmel-isi.c |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/drivers/media/video/atmel-isi.c b/drivers/media/video/atmel-isi.c
index 6274a91..87d2748 100644
--- a/drivers/media/video/atmel-isi.c
+++ b/drivers/media/video/atmel-isi.c
@@ -883,6 +883,12 @@ static int isi_camera_set_bus_param(struct soc_camera_device *icd)
 	return 0;
 }
 
+
+static int isi_camera_set_parm(struct soc_camera_device *icd, struct v4l2_streamparm *parm)
+{
+	return 0;
+}
+
 static struct soc_camera_host_ops isi_soc_camera_host_ops = {
 	.owner		= THIS_MODULE,
 	.add		= isi_camera_add_device,
@@ -894,6 +900,8 @@ static struct soc_camera_host_ops isi_soc_camera_host_ops = {
 	.poll		= isi_camera_poll,
 	.querycap	= isi_camera_querycap,
 	.set_bus_param	= isi_camera_set_bus_param,
+	.set_parm	= isi_camera_set_parm,
+	.get_parm	= isi_camera_set_parm,
 };
 
 /* -----------------------------------------------------------------------*/
-- 
1.7.10.4

